src/SitemapExtension.php

```diff
@@ -190,6 +190,23 @@ private function getLinks()
         return $links;
     }
 
+    /**
+     * @param string $contentTypeSlug
+     * @return string
+     */
+    private function getListingLink($contentTypeSlug)
+    {
+        $config = $this->getConfig();
+        $urlGenerator = $this->getContainer()['url_generator'];
+        $urlParameters = ['contenttypeslug' => $contentTypeSlug];
+
+        if(isset($config['listing_routes']) && isset($config['listing_routes'][$contentTypeSlug])) {
+            $routeName = $config['listing_routes'][$contentTypeSlug];
+            return $urlGenerator->generate($routeName, $urlParameters);
+        }
+        return $urlGenerator->generate('contentlisting', $urlParameters);
```

Comment:
It would be good to have empty line before return

Reply:
Fixed